cmd/root.go

```diff
@@ -155,6 +156,33 @@ func schedulePeriodicScan(ctx context.Context) func() error {
 	}
 }
 
+func schedulePeriodicBackup(ctx context.Context) func() error {
+	return func() error {
+		schedule := conf.Server.Backup.Schedule
+		if schedule == "" {
+			log.Warn("Periodic backup is DISABLED")
+			return nil
+		}
+
+		database := db.Db()
+		schedulerInstance := scheduler.GetInstance()
+
+		log.Info("Scheduling periodic backup", "schedule", schedule)
+		err := schedulerInstance.Add(schedule, func() {
+			start := time.Now()
+			err := database.Backup(ctx)
```

Comment:
Can't we call Prune here, and remove the Bypass flag? I don't like having attributes in the config that are not really config options. It could be injected somehow in the Db, but I think it is easier and clearer if we just call Prune here.